### DISCUSSION TITLE
STspin32G4 support

### DISCUSSION THREAD

**Discussion by Juanduino** [stm32duino/Arduino_Core_STM32#1714] [General]
Hei

Just came across this STspin32G4 which is a obvious candidate for [SimpleFoc ](https://community.simplefoc.com/t/stspin32g4-support/2027)(BLDC ESC)

I do not see the MCU in your list (STM32G431VBx3)?

**Comment by fpistm** [comment]
Hi @Juanduino 

Not all MCU have been added. But we provide all generic files to ease addition:
https://github.com/stm32duino/Arduino_Core_STM32/tree/main/variants/STM32G4xx/G431V(6-8-B)T_G441VBT

Hereafter the Wiki: 
https://github.com/stm32duino/wiki/wiki/Add-a-new-variant-%28board%29

**Comment by Juanduino** [comment]
I see, Thx!

If it happens, I’ll make sure to add a new variant. This is still in
conceptual dev.

All the best



man. 9. maj 2022 kl. 17.05 skrev Frederic Pillon ***@***.***>:
